File: src/main/java/Level1/Ex7.java
package Level1;

// 문자열 내 p와 y의 개수
public class Ex7 {
    boolean solution(String s) {
        boolean answer = true;
        int pCount = 0;
        int yCount = 0;
        System.out.println("Hello Java");
        for (int i = 0; i < s.length(); i++) {
            if ('p' == (s.charAt(i)) || 'P' == (s.charAt(i))) {
                pCount++;
            }
            if ('y' == (s.charAt(i)) || 'Y' == (s.charAt(i))) {
                yCount++;
            }
        }
        if (pCount == yCount) {
            answer = true;
        } else {
            answer = false;
        }
        return answer;
    }

    public static void main(String[] args) {
        Ex7 ex7 = new Ex7();
        String s = "pPoooyY";
        System.out.println(ex7.solution(s));
    }
}

File: src/main/Javascript/Level1/Ex7.js
function solution(s) {
    var answer = true;
    let str = s.split("");
    let pCount = 0;
    let yCount = 0;
    for (let i = 0; i < str.length; i++) {
        if (str[i] === 'p' || str[i] === 'P') {
            pCount++;
        }
        if (str[i] === 'y' || str[i] === 'Y') {
            yCount++;
        }
    }
    if (pCount !== yCount) {
        answer = false;
    }
    return answer;
}

console.log(solution("pPoooyY"));
console.log(solution("Pyy"));